src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.story.js.bckp

```diff
@@ -45,4 +47,14 @@ storiesOf('Components/Site Headers/Harmonised', module)
     {
       notes: { markdown: notes, json: dataGroup2 },
     }
+  )
+  .add(
+    'group 3',
+    () =>
+      siteHeaderHarmonised(
+        merge(dataGroup3, {})
```

Comment:
What is this..? :)
Why merging the specs with an empty object? We use the merge when we want to pass additional data or we want to override some of the existing, in this case i see we don't need anything, instead.